### DISCUSSION TITLE
downloading

### DISCUSSION THREAD

**Discussion by potatos69420** [KSP-CKAN/CKAN#3718] [Q&A] [answered]
how the hell do i use this, i have no idea how to download it

**Comment by HebaruSan** [comment, marked as answer]
There is a big download link in the readme on the main page:

![image](https://user-images.githubusercontent.com/1559108/202890252-06aef680-011f-487d-a6b6-020abd4932bc.png)

https://github.com/KSP-CKAN/CKAN

**Reply by potatos69420** [reply]
thanks im not smort so

**Reply by potatos69420** [reply]
i clcked download and it didnt work

**Reply by HebaruSan** [reply]
It probably did.

**Reply by potatos69420** [reply]
my browser i use pops up a huge thing on my screen when i download stuff so it didnt for some reason

**Reply by HebaruSan** [reply]
Maybe you should read what the screen says after you click.

**Reply by potatos69420** [reply]
Nvm
github downloads it though itself

**Comment by HebaruSan** [comment]
And for using it, the user guide is here:

https://github.com/KSP-CKAN/CKAN/wiki/User-guide
